cypress/integration/dryRunner/reset.js

```diff
@@ -45,6 +50,66 @@ describe('Dry-runner — Reset', () => {
       .should('contain', 1)
   })
 
+  it('should be able to reset a game with knight mana cost modifier', () => {
```

Comment:
I think we should move this test (and all the Brawl tests) to `dryRunner/brawl.js`. What do you think? It’s no longer very relevant to the reset. We could keep a super simple test here that just makes sure that the game is reseted when changing the brawl modifier.